CONTRIBUTING.md

```diff
@@ -113,6 +115,22 @@ install it and then run with:
 cargo deny --all-features check all
 ```
 
+## Other implementations
+
+The Coreutils has different implementations, with different levels of completions:
+
+* [GNU's](https://git.savannah.gnu.org/gitweb/?p=coreutils.git)
+* [BSD](https://github.com/openbsd/src/tree/master/bin)
```

Comment:
I think we should note here that this is OpenBSD (and we should probably also mention FreeBSD).